client/lib/purchases/index.js

```diff
@@ -204,6 +212,19 @@ function hasCreditCardData( purchase ) {
 	return Boolean( purchase.payment.creditCard.expiryMoment );
 }
 
+/**
+ * Checks whether the purchase is capable of explicit renew.
```

Comment:
I think the repetitive use of "explicit" needs explanation. What about changing the whole comment to `Checks whether the purchase can be renewed by user interaction. This is necessary because some purchases (eg, .fr domains) can only be auto-renewed.`